src/main/java/net/sf/jabref/gui/maintable/MainTableColumn.java

```diff
@@ -144,4 +132,26 @@ public JLabel getHeaderLabel() {
             return new JLabel(getDisplayName());
         }
     }
+
+    public boolean isResolved(BibEntry entry) {
```

Comment:
I'm trying to understand why one needs a flag if the entry is resolved at all. Don't I need to in a field-bases?